src/main/java/Duke.java

```diff
@@ -6,5 +8,98 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        greeting();
+    }
+
+    public static void showList(Todo list[], int x){
+        if(x == 0)  System.out.println("\n____________________________________________________________");
```

Comment:
Kindly take note of brackets after the if statement and the spacing between "{" and the method name